systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -204,18 +203,18 @@ void testBuildFailsWithWrongChecksumOfArtifact() {
         KafkaConnectUtils.waitUntilKafkaConnectStatusConditionContainsMessage(testStorage.getNamespaceName(), testStorage.getClusterName(), "The Kafka Connect build failed(.*)?");
 
         LOGGER.info("Checking if KafkaConnect status condition contains message about build failure");
-        KafkaConnect kafkaConnect = KafkaConnectResource.kafkaConnectClient().inNamespace(testStorage.getNamespaceName()).withName(testStorage.getClusterName()).get();
+        KafkaConnect kafkaConnect = CrdResourceClients.kafkaConnectClient().inNamespace(testStorage.getNamespaceName()).withName(testStorage.getClusterName()).get();
 
         LOGGER.info("Deploying network policies for KafkaConnect");
-        NetworkPolicyResource.deployNetworkPolicyForResource(kafkaConnect, KafkaConnectResources.componentName(testStorage.getClusterName()));
+        NetworkPolicyUtils.deployNetworkPolicyForResource(kafkaConnect, KafkaConnectResources.componentName(testStorage.getClusterName()));
 
         Condition connectCondition = kafkaConnect.getStatus().getConditions().stream().findFirst().orElseThrow();
 
         assertTrue(connectCondition.getMessage().matches("The Kafka Connect build failed(.*)?"));
         assertThat(connectCondition.getType(), is(NotReady.toString()));
 
         LOGGER.info("Replacing plugin's checksum with right one");
-        KafkaConnectResource.replaceKafkaConnectResourceInSpecificNamespace(testStorage.getNamespaceName(), testStorage.getClusterName(), kC -> {
+        KafkaConnectUtils.replaceKafkaConnectInNamespace(testStorage.getNamespaceName(), testStorage.getClusterName(), kC -> {
```

Comment:
```suggestion
        KafkaConnectUtils.replaceInNamespace(testStorage.getNamespaceName(), testStorage.getClusterName(), kC -> {
```
as the CR name is stated in utils class name....but I am not sure about suggested change. Just an idea for discussion.

Reply:
Good point, I was thinking about it as well. I changed it everywhere else.

Reply:
> no secrets or connector modifications that would make this name dubious?

`KafkaConnectUtils` class doesn't contain any other replace method. For KafkaConnector replace method we have the KafkaConnectorUtils class (where the replace method anyway is).

And yeah, I changed it everywhere else. I was thinking about it before and made sense to me even before. For other resources we anyway have utils classes as well, where we can fit those methods.